tools/test-generator.py

```diff
@@ -32,8 +32,10 @@
         lon = lon1 + (lon2 - lon1) * j / count
         points.append((lat, lon))
 
-def send(lat, lon, course):
+def send(lat, lon, course, alarm):
     params = (('id', id), ('timestamp', int(time.time())), ('lat', lat), ('lon', lon), ('bearing', course))
+    if alarm:
+        params = params + (('alarm', 'sosAlarm'),)
```

Comment:
It is here. Please read original comments. I already did a code review and you basically ignored half of my comments.